### PR TITLE
healthcheck library update and a bit of dropwizard, affinity and mem

### PR DESCRIPTION
**Pull request by kovacshuni**
healthcheck library update and a bit of dropwizard, affinity and memory limits.

### CODE REVIEW

File: helm/methode-article-internal-components-mapper/values.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ replicaCount: 2
...
   repository: coco/methode-article-internal-components-mapper
   pullPolicy: IfNotPresent
-#resources:
-#  limits:
-#    memory: 256Mi
+resources:
+  requests:
+    memory: 128Mi
+  limits:
+    memory: 256Mi
```
Comment (tosan88) [inline helm/methode-article-internal-components-mapper/values.yaml]: Could you please increase these values? As per the grafana metrics it uses at least 340Mi

Comment (kovacshuni) [reply helm/methode-article-internal-components-mapper/values.yaml]: done

File: methode-article-internal-components-mapper.yaml [hunk 2]
```diff
@@ -194,4 +194,4 @@ validationConfiguration:
 
 appInfo:
     systemCode: "MethodeArticleInternalComponentsMapperApplication"
```
Comment (khatton-ft) [inline methode-article-internal-components-mapper.yaml]: Dewey has the system code `up-maicm`

Comment (kovacshuni) [reply methode-article-internal-components-mapper.yaml]: System code was updated, but somehow there was a conflict.
Anyways it should be all right now, please check.

**Pull request merged**